notebooks/4_SparseKernelMethods.ipynb

```diff
@@ -973,9 +982,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# The Utility Classes\n",
+    "# Implementation in `skcosmo`\n",
     "\n",
-    "Classes from the utility module enable computing SparseKPCA, SparseKRR, and SparseKPCovR with a scikit.learn-like syntax. `SparseKPCovR` is located in `utilities/kpcovr.py`."
+    "Classes from the skcosmo module enable computing SparseKPCA, SparseKRR, and SparseKPCovR with a scikit.learn-like syntax. `SparseKPCovR` is located in `skcosmo/kpcovr.py`."
```

Comment:
You'll find the actual location in https://github.com/cosmo-epfl/sklearn-cosmo/issues/3